src/component/routers/GroupsRouter.tsx

```diff
@@ -58,22 +59,22 @@ export class GroupsRouter extends React.PureComponent<GroupsRouterProps> {
         }
 
         const groupProps: Partial<GroupsElementProps> = {
-            localGroups: this.props.groups
+            localGroups: this.props.localGroups
         };
 
-        if (groupSplitExistsAtNode(this.props.nodeToEdit)) {
+        if (hasGroupsRouter(this.props.nodeToEdit)) {
```

Comment:
changed this because i realize `split` isn't used much; seems like we're favoring `router` throughout the repo